lib/ansible/utils/display.py

```diff
@@ -534,5 +534,6 @@ def _set_column_width(self):
         if os.isatty(1):
             tty_size = unpack('HHHH', fcntl.ioctl(1, TIOCGWINSZ, pack('HHHH', 0, 0, 0, 0)))[1]
         else:
-            tty_size = 0
+            # Avoid wrapping in absence of a tty
+            tty_size = sys.maxsize
```

Comment:
I'd argue that it would make more sense to make this infinity:
```suggestion
            tty_size = float('inf')
```

Reply:
I am not sure floats are allowed, an infinite float may have unexpected results IMHO.

Reply:
No, all's checked. And semantically, it'd be better than "an arbitrary big number".
```python
In [1]: float('inf') - 1
Out[1]: inf

In [2]: min(float('inf'), 79)
Out[2]: 79

In [3]: import textwrap

In [4]: textwrap.wrap('asdfa', float('inf'))
Out[4]: ['asdfa']
```